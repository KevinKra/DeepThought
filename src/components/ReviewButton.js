import React from "react";

const ReviewButton = props => {
  return <button type="submit">{props.purpose}</button>;
};

export default ReviewButton;
