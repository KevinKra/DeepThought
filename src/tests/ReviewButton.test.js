import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ReviewButton from "../components/ReviewButton";

const mockPurpose = "questionCard";

describe("<ReviewButton />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ReviewButton />);
  });
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
