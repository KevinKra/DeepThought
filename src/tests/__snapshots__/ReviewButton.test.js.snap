// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewButton /> should match the snapshot 1`] = `
<button
  type="submit"
/>
`;
